### DISCUSSION TITLE
Custom mask for all country

### DISCUSSION THREAD

**Discussion by jovanhartono** [goveo/react-international-phone#65] [Q&A] [answered]
It is possible to add custom masks for all countries? 
ex: +1 123-4567-8901, +2 123-4567-8901, etc..

nice project btw

**Comment by ybrusentsov** [comment, marked as answer]
@jovanhartono Thanks!
You can pass the country param with modified `format` country values:

```tsx
const countries = defaultCountries.map((country) => {
  const parsedCountry = parseCountry(country);

  return buildCountryData({
    ...parsedCountry,
    format: '...-....-....',
  });
});

export const App = () => {
  return <PhoneInput countries={countries} defaultCountry="ua" />;
};
```

But this example doesn't handle dial-code value, so this will be supported:
- +1 123-4567-8901
- +43 123-4567-8901
- +380 123-4567-8901
